src/main/java/frc/robot/subsystems/flywheel/factory/FlywheelFactory.java

```diff
@@ -8,8 +8,14 @@
 
 public class FlywheelFactory {
 
-	private static Flywheel
-		generateReelFlywheel(String topLogPath, String bottomLogPath, SparkMaxDeviceID topID, SparkMaxDeviceID bottomID, Robot robot) {
+	//@formatter:off
+	private static Flywheel generateReelFlywheel(
+			String topLogPath,
+			String bottomLogPath,
+			SparkMaxDeviceID topID,
+			SparkMaxDeviceID bottomID,
+			Robot robot
```

Comment:
you should only use one tab, not two
```suggestion
		String topLogPath,
		String bottomLogPath,
		SparkMaxDeviceID topID,
		SparkMaxDeviceID bottomID,
		Robot robot
```